src/main/java/com/pikachu/purple/bootstrap/common/config/SwaggerConfig.java

```diff
@@ -1,33 +1,34 @@
 package com.pikachu.purple.bootstrap.common.config;
 
-import io.swagger.v3.oas.annotations.OpenAPIDefinition;
-import io.swagger.v3.oas.annotations.info.Info;
-import io.swagger.v3.oas.annotations.servers.Server;
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.info.Info;
+import io.swagger.v3.oas.models.servers.Server;
 import org.springdoc.core.models.GroupedOpenApi;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
-
-@OpenAPIDefinition(
-    info = @Info(
-        title = "PERPICKS",
-        description = "Perpicks의 API 문서입니다.",
-        version = "v1"
-    ),
-    servers = @Server(description = "dev-server")
-)
 @Configuration
 public class SwaggerConfig {
 
+    @Value("${api.server-uri}")
+    private String serverUrl;
+
     @Bean
     public GroupedOpenApi allOpenApi() {
-        String[] paths = {"/perpicks/**"};
-
-        return GroupedOpenApi
-            .builder()
+        return GroupedOpenApi.builder()
             .group("전체 API")
-            .pathsToMatch(paths)
+            .pathsToMatch("/perpicks/**")
             .build();
     }
 
+    @Bean
+    public OpenAPI customOpenAPI() {
+        return new OpenAPI()
+            .info(new Info().title("PURPLE")
+                .description("Purple의 API 문서입니다.")
```

Comment:
p3) 이름 변경해주시면 감사하겠습니다!
```suggestion
            .info(new Info().title("PERPICKS")
                .description("Perpicks의 API 문서입니다.")
```

Reply:
오호 좋습니다!